swift/apple/README.md

```diff
@@ -2,33 +2,52 @@
 
 Firezone app clients for macOS and iOS.
 
-## Builidng
+## Pre-requisites
 
-Clone this repo:
+  - Rust
 
-```bash
-git clone https://github.com/firezone/firezone
-```
+## Building
 
-Build Connlib:
-```bash
-cd rust/connlib/clients/apple
-PLATFORM_NAME=macosx ./build-rust.sh # For macOS
-PLATFORM_NAME=iphoneos ./build-rust.sh # For iOS
-./build-xcframework-dev.sh
-```
+ 1. Clone this repo:
+
+    ```bash
+    git clone https://github.com/firezone/firezone
+    ```
+
+ 2. `cd` to the Apple clients code
+
+    ```bash
+    cd swift/apple
+    ```
+
+ 3. Rename and populate developer team ID file:
 
-Rename and populate developer team ID file:
+    ```bash
+    cp Firezone/Developer.xcconfig.template Firezone/Developer.xcconfig
+    vim Firezone/Developer.xcconfig
```

Comment:
It looks like the path for this has changed:
```suggestion
    cp Firezone/xcconfig/Developer.xcconfig.template Firezone/xcconfig/Developer.xcconfig
    vim Firezone/xcconfig/Developer.xcconfig
```

Reply:
Good catch. Fixed in [730081f](https://github.com/firezone/firezone/pull/1759/commits/730081f27ca1afcb420fe54f5db199f8a5de8837).